src/DynamoCoreWpf/ViewModels/Preview/InfoBubbleViewModel.cs

```diff
@@ -153,12 +416,27 @@ public InfoBubbleViewModel(DynamoViewModel dynamoViewModel)
             ConnectingDirection = Direction.None;
             Content = string.Empty;
             DocumentationLink = null;
-            ZIndex = 3;
+            // To appear above any NodeView elements
```

Comment:
Just curious, how to find this fact out?

Reply:
There was a comment I read somewhere; I think groups were at 0, connectors at 1 and node bodies at 2. I may be wrong here though.